flowclient/flowclient/client.py

```diff
@@ -455,6 +455,320 @@ def daily_location(
     }
 
 
+def _meaningful_locations(
+    start_date: str,
+    stop_date: str,
+    label: str,
+    labels: Dict[str, Dict[str, dict]],
+    tower_cluster_radius: float = 1.0,
+    tower_cluster_buffer: float = 0.0,
+    tower_cluster_call_threshold: int = 0,
+    tower_dow_scores: Dict[str, float] = {
```

Comment:
Personally, I always struggle to interpret variable names that have a mix of full words and abbreviations in them.

I think I'd be in favour of a longer but more explicit name such as `tower_day_of_week_scores` (or perhaps `tower_weekday_scores`), but I realise this is personal preference so I'm also ok to leave it as it is if you prefer.